### PR TITLE
Accommodation search form

### PR DESCRIPTION
**Pull request by nazmuldipu**
Accommodation search form implemented

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
   <body>
     <div id="app">Home page</div>
     <a href="/calendar.html">Date Range Picker</a>
+    <a href="http://localhost:3000/accommodation-search-form/examples/accommodation-search-form.html">Accomation Seach Form</a>
```
Comment (xianmalik) [inline index.html]: Should this be localhost link? or relative link?

File: accommodation-search-form/examples/accommodation-search-form.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                endDate.setDate(endDate.getDate() + 2);
+                // picker.setDateRange(startDate, endDate);
+                startDate.setDate(startDate.getDate() + 1);
+                picker.selectStartDate(startDate);
+                picker.selectEndDate(endDate);
+                picker.goToNextMonth();
+                picker.goToPrevMonth();
+                picker.setPriceList(prices);
+                console.log(picker.getCurrentDateRange());
+                console.log(picker.getCurrentEndDate());
```
Comment (xianmalik) [inline accommodation-search-form/examples/accommodation-search-form.html:131]: Console logs present here

File: accommodation-search-form/accommodation-search-form.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+        </select>
+    </div>`;
+};
+const renderGuest = () => {
```
Comment (azs06) [inline accommodation-search-form/accommodation-search-form.js]: I feel like this is something should be configureable, for example someone might allow more than 10 guests, some might have limitations of 5 at once.

**Review by azs06 — CHANGES_REQUESTED** [review]

**Review by azs06 — APPROVED** [review]

**Comment by azs06** [comment]
**Note: need to separate utility.js file**

**Pull request merged**